### PR TITLE
Removed meaningless commens & added field mapping

### PR DESCRIPTION
**Pull request by HirokazuNishi**
- Removed meaningless comments from some files.
- added "to_customer_address" aspect for kana fields.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2-jp?pullRequest=45) <br/>All committers have signed the CLA.

File: Zip2address/README.md [hunk 1]
```diff
@@ -1,61 +0,0 @@
-Japanese Postcode Address converter
-================
```
Comment (ishakhsuvarov) [inline Zip2address/README.md:2]: Did this Readme become obsolete?

Comment (HirokazuNishi) [reply Zip2address/README.md:2]: I think... it should be rewritten completely.

**Comment by ishakhsuvarov** [comment]
@HirokazuNishi Thank you

**Pull request merged**